exercises/raindrops/uRaindropsTest.pas

```diff
@@ -0,0 +1,179 @@
+unit uRaindropsTest;
+
+interface
+uses
+  DUnitX.TestFramework;
+
+type
+
+  [TestFixture]
+  RaindropsTest = class(TObject)
+  public
+    [Test]
+    procedure the_sound_for_1_is_1;
+
+    [Test]
+    [Ignore('Comment these lines to run these tests')]
```

Comment:
But then, that is a generated exercise, and so it appears there.  This is not necessarily the case for the ones that are still not generated.  Just trying to organize so that any of the tests that we start generating with has a good 'baseline".